skypy/utils/completeness_function.py

```diff
@@ -0,0 +1,43 @@
+"""Completeness functions.
+
+This module computes completeness functions of cosmological surveys.
+
+.. autosummary::
+   :nosignatures:
+   :toctree: ../api/
+
+   logistic_completeness_function
+
+"""
+
+import numpy as np
+
+
+def logistic_completeness_function(m, m95, m50):
+    r'''Compute logistic completeness function.
+
+    This function calculates the logistic completeness function
+
+    .. math::
+
+        p(m) = \frac{1}{1 + \exp\left[\frac{\ln(1/19)}{m_{95} - m_{50}} (m - m_{50})\right]}\;.
+
```

Comment:
```suggestion
    .. math::

        p(m) = \frac{1}{1 + e^{\kappa (m - m_{50})}}
        
     with :math:`\kappa = \frac{\ln(1/19)}{m_{95} - m_{50}}`.

```
You could try make it more readable maybe?